herast/tree/patterns/expressions.py

```diff
@@ -303,6 +273,18 @@ def children(self):
 		return (self.first_operand, self.second_operand)
 
 
+class AbstractMemOpPat(ExpressionPat):
```

Comment:
I think name could be shorter, I like MemberPat. Also this is not quite an expression pattern, it belong to patterns/helpers.py logically

Reply:
I thought that the helpers were for logical things, things like and all of them between expression or any of them and not for decompiler operations such as memref and memptr

Reply:
Abstraction of member expressions is a logical thing... There are other helpers, that are pretty similar (stringpat, intpat). I even found a good example :) https://github.com/Mizari/herast/blob/main/herast/tree/patterns/helpers.py#L113

Reply:
I used it only to be able to generate the 2 classes that were the same except the op, I think it should be removed if you split them back with the fix with the different the doc string

Reply:
I didn't think of it as an helper because then it would be the same as StructFieldAccessPat the way I implemented AbstractMemOpPat, just wanted to fix the 2 of them by writing one line of code